ClientApp/viking-vault/src/ExchangeForm.tsx

```diff
@@ -233,15 +238,18 @@ class ExchangeForm extends React.Component<any, IExchangeFormState> {
                     body: JSON.stringify([  
                         {  
                             "CurrencyType": this.state.fromCurrency,
```

Comment:
No need to make the properties of the JSON object PascalCase as .NET automatically does that for you

Reply:
sooo, should it be "currencyType"?

Reply:
Yeah, pretty much, and when the data is mapped to C# objects, it will be accessible as CurrencyType

Reply:
Changed it, thank you!